react-backend/package.json

```diff
@@ -10,6 +10,7 @@
     "debug": "~2.6.9",
     "express": "~4.16.1",
     "http-errors": "~1.6.3",
+    "jade": "^1.11.0",
```

Comment:
I'll remove this with a dedicated PR to clean up the back end

Reply:
ok, no problemo